sematic/ui/packages/common/src/pages/RunDetails/RunTreeSection.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+import Typography from '@mui/material/Typography';
+
+import theme from 'src/theme/new';
+import RunTree from 'src/component/RunTree';
+
+const Section = styled.section`
+    display: flex;
+    flex-direction: column;
+    margin-bottom: ${theme.spacing(3)};
+}
+`
+const Headline = styled.h2`
+    margin-bottom: ${theme.spacing(3)};
+`;
+
+const StyledTypography = styled(Typography)`
+    height: ${theme.spacing(10)};
+    align-items: center;
+    display: flex;
+`;
+
+const RunTreeSection = () => {
+    const nodes = [
+        {value: 'MNIST PyTorch Example', children: [
+            {value: 'Load train dataset', children: [] as any},
+            {value: 'Load test dataset', children: [] as any},
+            {value: 'get_dataloader', children: [] as any},
+            {value: 'get_dataloader', children: [] as any},
+            {value: 'train_eval', children: [
+                {value: 'train_model', children: [] as any},
+                {value: 'evaluate_model', selected: true, children: [] as any}
```

Comment:
Looking at the storybook, this one seems to have a inconsistent width.
![Screenshot 2023-03-23 at 10 26 51 AM](https://user-images.githubusercontent.com/429433/227287851-07767601-9ee8-4b2a-8174-6e4c3a3f8720.png)